source/CardGames.py

```diff
@@ -438,24 +558,76 @@ def winnnerCheck(self, house, player):
       print("The house has blackjack. You lose!")
       betting_box.collect(blackjackOutcome)
       print(f"You now have ${player_name.money}")
+
```

Comment:
handsDisplay() should be handsDisplay(False) so the dealer's hand isn't hidden and the player can be sure that it was blackjack. 

The Y coordinate is too low in screen.blit(promptText, (350,400)) and the pomptText is being hidden by the player's cards. Something like this works better: screen.blit(promptText, (350,385))

Reply:
good catches both should now be fixed